### DISCUSSION TITLE
How to setup RustDesk Server on my Windows 7 through the internet after mapping it through SakuraFrp ?

### DISCUSSION THREAD

**Discussion by agilersu** [rustdesk/rustdesk#8771] [Q&A] [answered]
How to setup RustDesk Server on my Windows 7 through the internet after mapping it through SakuraFrp ?
I have installed RustDesk Server on my Windows 7 64-bit Ultimate edition and successfully started hbbs and hbbr. I can access it remotely within my intranet using its ID and relay server.

Now I want to access my Windows 7 machine remotely using its ID and relay server through the internet after mapping it through SakuraFrp. Could you please advise on what ports I should set up? My Windows 7 firewall is already disabled. Additionally, what other settings need to be adjusted in the RustDesk Server application on Windows 7?

**Comment by deep-soft** [comment]
Straight to the point:
https://pedja.supurovic.net/setting-up-self-hosted-rustdesk-server-on-windows/

More here:
https://github.com/rustdesk/rustdesk/wiki/FAQ
and
https://github.com/rustdesk/doc.rustdesk.com

**Reply by agilersu** [reply]
Many thanks! I found the points are very complex. May I get the screen shot of the operation process ?

**Comment by xlionjuan** [comment, marked as answer]
Please don't, unless you accept the risk of using EOL Windows.
